cloudformation/aws-ec2-vpcendpoint.go

```diff
@@ -10,6 +10,11 @@ import (
 // See: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html
 type AWSEC2VPCEndpoint struct {
 
+	// IsPrivateDnsEnabled AWS CloudFormation Property
+	// Required: false
+	// See: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html#cfn-ec2-vpcendpoint-isprivatednsenabled
+	IsPrivateDnsEnabled bool `json:"IsPrivateDnsEnabled,omitempty"`
```

Comment:
This should have been `PrivateDnsEnabled`, at least based on the CF docs https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html#cfn-ec2-vpcendpoint-privatednsenabled